src/scales/scale.time.js

```diff
@@ -45,8 +58,17 @@ module.exports = function(Chart) {
 				throw new Error('Chart.js - Moment.js could not be found! You must include it before Chart.js to use the time scale. Download at https://momentjs.com');
 			}
 
+			this.mergeTicksOptions();
+
 			Chart.Scale.prototype.initialize.call(this);
 		},
+		mergeTicksOptions: function() {
```

Comment:
This method should apply to all scale, so the new `major` options can be used by other scale implementations. That's why this merging logic should be in the base `core.scale.js`.